tests/test_plot.py

```diff
@@ -522,3 +539,28 @@ def test_corner_plot_fields_exclude_error(live_points):
     with pytest.raises(ValueError) as excinfo:
         plot.corner_plot(live_points, include=["x"], exclude=["logL"])
     assert "Cannot specify both `include` and `exclude`" in str(excinfo.value)
+
+
+def test_corner_plot_save_error(caplog, live_points):
+    """Assert a warning is printed if the figure cannot be saved"""
+    with patch.object(
+        mpl.figure.Figure,
+        "savefig",
+        side_effect=ValueError,
+    ) as mock_save:
+        plot.corner_plot(live_points, filename="corner.png")
+    mock_save.assert_called_once()
+    assert "Could not save test plot" in str(caplog.text)
```

Comment:
Should it be "could not save corner plot"?

Reply:
Indeed! I'll fix that now